apps/admin-gui/src/app/shared/components/dialogs/create-resource-dialog/create-resource-dialog.component.html

```diff
@@ -10,10 +10,16 @@
 
     <mat-form-field>
       <input matInput required placeholder="{{'DIALOGS.CREATE_RESOURCE.NAME' | translate}}" [formControl]="nameCtrl">
+      <mat-error>
+        {{'DIALOGS.CREATE_RESOURCE.FILL_NAME' | translate}}
```

Comment:
wrong label for this
also I would rather prefer if you use the markAsTouched in ngOnInit in this dialog